anchore_engine/services/catalog/catalog_impl.py

```diff
@@ -2541,4 +2542,33 @@ def add_event_json(event_json, dbsession, quiet=True):
             raise
 
 
+def load_policy_bundles(config, process_bundle, process_exception):
```

Comment:
Per comment above, let's move this to a general location. With the callback it seems able to be moved to a general spot.